Editor/Windows/AssetPaletteWindowEntryPanel.cs

```diff
@@ -562,14 +562,27 @@ private void DeselectEntry(PaletteEntry entry)
 
         private void SelectEntry(PaletteEntry entry, bool exclusively)
         {
+            if (PaletteEntry.IsEntryBeingRenamed && PaletteEntry.EntryCurrentlyRenaming != entry)
+            {
+                PaletteEntry.CancelRename();
+                renameText = string.Empty;
```

Comment:
Maybe `renameText = string.Empty` can be moved inside `CancelRename` to avoid duplication?

Reply:
Sure thingo!

Reply:
Wait, `renameText` is local :(

Reply:
Could make an `isCancel` parameter for `AssetPaletteWindowEntryPanel`'s `StopEntryRename` maybe, that one already calls `PaletteEntry.CancelRename`